### PR TITLE
300 frontend restyle landing section

### PR DESCRIPTION
**Pull request by rxia357**
landing page finished
added two new button styles(just bigger font size than default)
made responsive
merged with current main

### CODE REVIEW

File: package.json
Comment (bcho892) [inline package.json:1]: Can we revert changes in this file?

File: package-lock.json
Comment (bcho892) [inline package-lock.json:1]: This file should be deleted

File: package.json
Comment (bcho892) [reply package.json:1]: `git checkout master package.json`

File: client/src/assets/logos/FacebookBlue.png
Comment (bcho892) [inline client/src/assets/logos/FacebookBlue.png:1]: is there any way we can use svgs for the logos instead?

Comment (bcho892) [reply client/src/assets/logos/FacebookBlue.png:1]: Check how it was done in the footer

File: yarn.lock
[the remark this answers is not in the document]
Comment (rxia357) [reply yarn.lock:1]: yep changed have been reverted

File: package-lock.json
Comment (rxia357) [reply package-lock.json:1]: deleted

File: package.json
Comment (rxia357) [reply package.json:1]: done

**Comment by bcho892** [comment]
Screenshot_20240531-210946
Double check on mobile screens - it seems like there is excessive paddingg or margins

**Review by bcho892 — APPROVED** [review]

**Pull request merged**